src/utils/labels.js

```diff
@@ -15,28 +15,38 @@ const fonts = {
 const getOffsetEms = (type, radius = 5, fontSize = 11) =>
     type === 'Point' ? radius / parseInt(fontSize, 10) + 0.4 : 0
 
-export const labelSource = (features, { fontSize }, isBoundary) => ({
-    type: 'geojson',
-    data: featureCollection(
-        features.map(({ geometry, properties }) => ({
-            type: 'Feature',
-            geometry: {
-                type: 'Point',
-                coordinates: getLabelPosition(geometry),
-            },
-            properties: {
-                name: properties.name,
-                anchor: geometry.type === 'Point' ? 'top' : 'center',
-                offset: [
-                    0,
-                    getOffsetEms(geometry.type, properties.radius, fontSize),
-                ],
-                color: isBoundary ? properties.color : '#333',
-                value: properties.value,
-            },
-        }))
-    ),
-})
+export const labelSource = (
```

Comment:
I think it would be good to add some jest tests for this function.

Reply:
Addressed here: https://github.com/dhis2/maps-gl/pull/596/commits/bb818748124ed19a517ac72123d8411edbf9e14f